src/code/z_lifemeter.c

```diff
@@ -156,17 +156,17 @@ void HealthMeter_Update(GlobalContext* globalCtx) {
 
     if (interfaceCtx) {}
 
-    if (interfaceCtx->unk_200 != 0) {
-        interfaceCtx->unk_1FE--;
-        if (interfaceCtx->unk_1FE <= 0) {
-            interfaceCtx->unk_1FE = 0;
-            interfaceCtx->unk_200 = 0;
+    if (interfaceCtx->beatingHeartChangeDirection) {
+        interfaceCtx->beatingHeartChangeFactor--;
```

Comment:
now that its direction we should add != 0 and change the true/falses. after that this is good from me